src/main/java/it/gov/pagopa/Entity/OrganizationEntity.java

```diff
@@ -0,0 +1,28 @@
+package it.gov.pagopa.Entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "organizations")
+public class OrganizationEntity extends PanacheEntityBase{
+    
+    @Id
+    @Column(name = "organizationFiscalCode")
+    private String organizationFiscalCode;
```

Comment:
adopt a numeric id instead of a string to improve performances and then set unique on organizationFiscalCode to prevent duplicate